### PR TITLE
Add compatibility for multiple S3 services

### PR DESCRIPTION
**Pull request by shumatepf**
More S3 services require access via the IDVA platform and the s3-interface app can only interface to a single S3 service. This PR contains the following changes to adjust to new requirements:

- Check for multiple S3 services in settings
- Specific S3 service that is being accessed added to path parameter `/get/<service_id>
- Switch from boto3 resources to clients

### CODE REVIEW

File: .github/workflows/deploy.yaml [hunk 1]
```diff
@@ -30,6 +30,8 @@ jobs:
           cf-username: ${{ secrets.CF_USERNAME }}
           cf-password: ${{ secrets.CF_PASSWORD }}
           cf-org: ${{ secrets.CF_ORG }}
+          VENDOR_1: ${{ secrets.VENDOR_1 }}
+          VENDOR_2: ${{ secrets.VENDOR_2 }}
 
       - name: Deploy application
         run: cf push --vars-file vars.yaml
```
Comment (dzaslavskiy) [inline .github/workflows/deploy.yaml:35]: Shouldn't the variable go in the deploy step?

File: manifest.yaml [hunk 2]
```diff
@@ -10,4 +10,5 @@ applications:
       - python_buildpack
     command: gunicorn s3.main:app
     services:
-      - sk-s3
+      - ((VENDOR_1))
```
Comment (dzaslavskiy) [inline manifest.yaml]: instead of calling them vendors, I would refer to them as services or buckets

File: .github/workflows/deploy.yaml (hunk 1 above)
Comment (shumatepf) [reply .github/workflows/deploy.yaml:35]: Good point, moved accordingly

File: manifest.yaml (hunk 2 above)
Comment (shumatepf) [reply manifest.yaml]: Renamed to services

**Review by dzaslavskiy — APPROVED** [review]

**Pull request merged**